src/mail/view/MailViewer.ts

```diff
@@ -678,9 +679,38 @@ export class MailViewer implements Component<MailViewerAttrs> {
 			})
 	}
 
+	private async getAvailableFolders(): Promise<MailFolder[]> {
```

Comment:
Change the name or document. What are the folders available for?

Reply:
I decided to just not extract this code since it's only used once.